### PR TITLE
Small fix

### PR DESCRIPTION
**Pull request by mmpagani**

### CODE REVIEW

File: openquake/hazardlib/sourcewriter.py [hunk 1]
```diff
@@ -444,6 +444,8 @@ def get_source_attributes(source):
     """
     attrs = {"id": source.source_id, "name": source.name}
     if isinstance(source, NonParametricSeismicSource):
+        if not hasattr(source.data[0][0], 'weight'):
+            return attrs
```
Comment (micheles) [inline openquake/hazardlib/sourcewriter.py:448]: Can you add a comment explaining when this happens?

Comment (mmpagani) [reply openquake/hazardlib/sourcewriter.py:448]: Hard to explain. It's occurring when creating the sources for the JPN model.

Comment (mmpagani) [reply openquake/hazardlib/sourcewriter.py:448]: The weight is actually something you use ;)

**Comment by micheles** [comment]
Superseded by https://github.com/gem/oq-engine/pull/8335

**Pull request closed without merging**